src/clients/HydrogenClient.ts

```diff
@@ -4,13 +4,17 @@ import { BlockchainUtils } from "@carbon-sdk/util";
 import { EnumMember } from "typescript";
 import { ChainTransaction, CrossChainTransfer, CrossChainTransferDetailed, GetDetailedTransfersResponse, GetStatsResponse, GetTransfersRequest, GetTransfersResponse } from "../hydrogen";
 import dayjs from "dayjs";
+import { FeeQuote, GetFeeQuoteRequest, GetFeeQuoteResponse } from "@carbon-sdk/hydrogen/feeQuote";
 
```

Comment:
Do add the Hydrogen Testnet URL to the networkConfig file (`src/constant/network.ts`). It should added to `hydrogenUrl` under the network config of `CarbonSDK.Network.Testnet`.
<img width="460" alt="Screenshot 2022-08-29 at 3 24 12 PM" src="https://user-images.githubusercontent.com/65583522/187146530-33121ff8-ce06-444e-a927-b7180f2fe05c.png">